client/ayon_harmony/plugins/create/create_render_layers.py

```diff
@@ -0,0 +1,1002 @@
+His head will be rendered too but without any other parts. The same for body
+and arm.
+
+What is this good for? Compositing has more power how the renders are used.
+Can do transforms on each render pass without need to modify a re-render them
+using Harmony.
+
+The workflow may hit issues when there are used other blending modes than
+default 'color' blend more. In that case it is not recommended to use this
+workflow at all as other blend modes may affect all layers in clip which can't
+be done.
+
+There is special case for simple publishing of scene which is called
+'render.scene'. That will use all visible layers and render them as one big
+sequence.
+
+Todos:
+    Add option to extract marked layers and passes as json output format for
+        AfterEffects.
+"""
+
+import re
+import collections
+from typing import Any, Optional, Union
+from dataclasses import dataclass
+
+from ayon_core.lib import (
+    AbstractAttrDef,
+    EnumDef,
+    BoolDef,
+)
+from ayon_core.pipeline.create import (
+    CreatedInstance,
+    CreatorError,
+    get_product_name
+)
+
+from ayon_harmony.api.plugin import HarmonyCreator, HarmonyRenderCreator
+from ayon_harmony.api.lib import get_layers_info
+import ayon_harmony.api as harmony
+
+
+RENDER_LAYER_DETAILED_DESCRIPTIONS = """Render Layer is "a group of Harmony layers"
+
+Be aware Render Layer <b>is not</b> Harmony layer.
+
+All Harmony layers in the scene with the color group id are rendered in the
+beauty pass. To create sub passes use Render Pass creator which is
+dependent on existence of render layer instance.
+
+The group can represent an asset (tree) or different part of scene that consist
+of one or more Harmony layers that can be used as single item during
+compositing (for example).
+
+In some cases may be needed to have sub parts of the layer. For example 'Bob'
+could be Render Layer which has 'Arm', 'Head' and 'Body' as Render Passes.
+"""
+
+
+RENDER_PASS_DETAILED_DESCRIPTIONS = """Render Pass is sub part of Render Layer.
+
+Render Pass can consist of one or more Harmony layers. Render Pass must
+belong to a Render Layer. Marked Harmony layers will change it's group color
+to match group color of Render Layer.
+"""
+
+
+AUTODETECT_RENDER_DETAILED_DESCRIPTION = """Semi-automated Render Layer and Render Pass creation.
+
+Based on information in Harmony scene will be created Render Layers and Render
+Passes. All color groups used in scene will be used for Render Layer creation.
+Name of the group is used as a variant.
+
+All Harmony layers under the color group will be created as Render Pass where
+layer name is used as variant.
+
+The plugin will use all used color groups and layers, or can skip those that
+are not visible.
+
+There is option to auto-rename color groups before Render Layer creation. That
+is based on settings template where is filled index of used group from bottom
+to top.
+"""
+
+@dataclass
+class GroupInfo:
+    id: str  # #ffffff
```

Comment:
Please use `color`. I know that you use it as id but for future readers (which is also you BTW) please be specific about it.